README.md

```diff
@@ -56,6 +56,9 @@ An ([IDE](https://en.wikipedia.org/wiki/Integrated_development_environment)) is
 - [HASH](https://hash.ai) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Build and run JavaScript, Python, or Rust multi-agent simulations in the browser. Provides visualization, analysis, and experiment features.
 - [goormIDE](https://ide.goorm.io) <img title="Sign Up Required" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/shopify.svg" /> - Run almost every programming languages with real-time collaboration, live pair programming, and built-in messenger. 
 - [Coding Rooms](https://codingrooms.com/compiler/) - Edit and run JavaScript, Java, Python, C++ and more in a collaborative environment with intellisense, live chat, audio and video conferencing, real-time editing, and terminal.
+- [Backlight](https://backlight.dev/) - Run and edit any language in Backlight, a saas coding platform implementing [Monaco Editor](https://microsoft.github.io/monaco-editor/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/Microsoft/monaco-editor) where front-end teams build, document, publish, scale and maintain Design Systems.
+- [Webcomponents.dev](https://webcomponents.dev/) - Run and edit any language in a saas IDE based on [Monaco Editor](https://microsoft.github.io/monaco-editor/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/Microsoft/monaco-editor) where you can code web components in isolation, visualize them in stories, test and publish them on npm.
+- [Components.studio](https://webcomponents.dev/) - Run and edit any language to code components in isolation, visualize them in stories, test and publish them on npm.
```

Comment:
Looks like you put webcomponents.dev twice

Reply:
Oops sorry I fixed it.